hang_man/main.js

```diff
@@ -0,0 +1,114 @@
+const mystery_words_array = [
+  "Malamute",
+  "Poodle",
+  "Labrador",
+  "Spaniel",
+  "Bloodhound",
+  "Sheppard",
+  "Weiner",
+  "Bulldog",
+  "Boxer",
+  "Chihuahua",
+  "Mutt",
+  "Weimaraner",
+  "Pug"
+]
+const mystery_word = mystery_words_array[Math.floor(Math.random()*mystery_words_array.length)].toLowerCase();
+console.log(mystery_word)
+const mystery_letter_array = mystery_word.split("");
+let wrong_attempt_counter = 0;
+let win = 0;
+const you_win_sound = new Audio(`sounds/win.wav`);
+const you_loose_sound = new Audio(`sounds/fart.wav`);
```

Comment:
nitpick: the file name should match the _purpose_ not the content. File should be `lose.wav`. If you ever want to change the sound (say, to a sad trombone) you can just replace the file, without having to change the code. Also, the file name gives away the surprise :wink: